pandas/core/ops.py

```diff
@@ -1049,6 +1082,20 @@ def na_op(x, y):
         return result
 
     def safe_na_op(lvalues, rvalues):
```

Comment:
Ideally I'd like to get rid of this altogether to de-nest the closures, catch a more specific exception, and do this directly in `na_op`

Reply:
sure, separate PR :>